zuul.d/jobs.yaml

```diff
@@ -22,10 +22,26 @@
     nodeset: centos-8-1vcpu
 
 - job:
-    name: ansible-tox-molecule
-    parent: tox-molecule
+    name: ansible-tox-docs
+    parent: tox-docs
+    nodeset: centos-8-1vcpu
+
+- job: &_stub_molecule
```

Comment:
moving to here, as this is the actual job.

I don't think I want to get into the idea of private / public jobs, while I understand the idea. Until now, with parent / child jobs' we've used the idea of 'base' to indicate a top-level job, where child to parent to.

This is a little different, as we are trying reduce copypasta in job design. While not fully documented, zuul jobs can have multiple parents, so I wonder if that might be something to try here.  Let me experiment with something in etherpad, so see if we can review all options.

Reply:
This would be the zuul way to do this, without using yaml anchors, which means a little more typing, but removes need for private jobs:

  https://etherpad.openstack.org/p/ansible-tox-molecule